### DISCUSSION TITLE
Kann ich irgendwie den Minimum SOC auslesen?

### DISCUSSION THREAD

**Discussion by akanezumi** [aruttkamp/ioBroker.rct#82] [Fragen & Antworten] [answered]
Kann ich irgendwie den Minimum SOC auslesen, unter welchen die Batterie im Normalfall nicht entladen wird (im Standard sind das meines Wissens 7%). Ich habe leider intuitiv keinen passenden Wert gefunden.

**Comment by aruttkamp** [comment]
Ja, kannst du ! 
Nutze die Parameter power_mng.soc_min und power_mng.soc_min_island
.......
und dann warte auf das Update des Adapters *hüstel*

**Comment by akanezumi** [comment]
power_mng.soc_min und power_mng.soc_min_island liefern bei mir 0,1 als Wert. Das sieht nicht korrekt aus. Soll der zweite Satz andeuten, dass der rct iobroker das aktuell nicht kann?

**Reply by aruttkamp** [reply, marked as answer]
Sorry. ich habe mich da unklar ausgedrückt. 
Ab Adapter Version 1.2.3 kann er das korrekt ausgeben. 
Die 0,1 sind ein gerundeter Wert... das ist an dieser Stelle quatsch. 
Adapter Version 1.2.4 ist soeben online bei NPM. Kannst entweder bis morgen warten ,dann zeigt ioBroker dir das Update oder du installerst über die Katze direkt aus NPM.

**Reply by akanezumi** [reply]
Mega. In 1.2.4 liefert power_mng.soc_min_island nun "7" was korrekt aussieht! Danke für den schnellen Support.

**Reply by akanezumi** [reply]
Gibt es irgendwo eine Übersicht/Dokument, in dem beschrieben ist, welche Parameter was genau darstellen? Ich habe teilweise etwas Probleme die Werte aus dem Adapter mit den Werten aus der RCT App oder dem Portal in Verbindung zu bringen. 

Beispiele: im Portal sehe ich einen Wert für "PV Erzeugung heute". Da finde ich keinen Parameter der einen ähnlichen Wert ausspuckt.

**Comment by NCIceWolf** [comment]
RCT ist da leider nicht ganz so toll mit der Dokumentation unterwegs.

In der rct_core2.js ist etwas weiter unten bei den "rct.cmdReverse" Werten bei einigen eine Beschreibung angehängt.
Meiner Erfahrung nach sind die Werte ohne Beschreibung generell unbrauchbar, aber genau deswegen steht in der Readme das es keine genaue Beschreibung gibt und die Nutzung von Werten abseits der Standards auf eigene Gefahr erfolgt ;-)
-- 
Gesendet mit pEp für Android.

**Comment by aruttkamp** [comment]
Schaut euch das hier mal an ;-) 
https://rctclient.readthedocs.io/en/latest/inverter_app_mapping.html

**Reply by NCIceWolf** [reply]
Na da steht ja das gleiche drin wie in der rct_core2.js ;)

**Reply by aruttkamp** [reply]
Abgesehen von den Screenshots aus der APP und der Erläuterung welcher Wert welcher ist - ja !

**Reply by NCIceWolf** [reply]
Man sollte das mal richtig lesen, das kannte ich noch gar nicht!
Ist ja doch mal eine wirklich sinnvolle Anleitung. Könnte man durchaus in der Readme verlinken wenn man wollen würde!

**Reply by aruttkamp** [reply]
Da würde ich eher was eigenes Bevorzugen. So verlinkte Sachen sind plötzlich weg und dann steht man da.

**Reply by akanezumi** [reply]
Vor allem stimmen die teilweise auch nicht. Zumindest bin ich gerade auf eine Ungereimtheit gestoßen. Laut der verlinkten Seite soll g_sync.p_ac_grid_sum_lp identisch mit dem Wert zum Netzbezug aus der App sein (Symbol 'K' auf der Grafik). Aber das passt nicht. Dieser Wert ist zumindest bei mir quasi nie negativ, was er nach meinem Verständnis bei Netzbezug aber sein sollte.
Ich habe in der Tat auch bisher keinen anderen Parameter gefunden, der mir in einem Wert Netzbezug bzw. Einspeisung darstellt.

**Reply by akanezumi** [reply]
Andere Frage: hat schon mal jemand bei RCT nachgefragt, ob man ein paar (mehr) Informationen bekommen kann?

**Reply by aruttkamp** [reply]
Also .. der Wert g_synx.p_ac_grid_sum_lp entspricht exakt dem Wert in der APP. 
Es werden sowohl positive wie auch negative Werte angezeigt. Das ganze ist nach meiner Einschätzung auch plausibel.

Ich vermisse ehrlich gesagt keine Informationen die ich mir von RCT holen könnte. Bisher habe ich immer alles gefunden ;-) 

Frage an Dich .. wie sieht deine Config des RCT Adapters aus ?

**Reply by akanezumi** [reply]
Interessant. Ich logge die Werte über iobroker in eine Influx Datenbank. Der Wert g_sync.p_ac_grid_sum_lp war noch nie nennenswert unter 0. Ein paar mal in den letzten 30 Tagen ist er tatsächlich ganz knapp unter 0 gesunken. Das sieht man auf einem Graphen allerdings nicht, daher ist es mir bisher noch nicht aufgefallen. 

Adapter Version: v1.2.4; Konfigurieren kann man ja ansonsten nicht viel.

**Reply by akanezumi** [reply]
Aktuell stimmt g_sync.p_ac_grid_sum_lp mit dem Wert in der App auch überein, aber ich produziere gerade auch genug. Ich werde das heute Abend, wenn's dunkel ist noch mal überprüfen, was in iobroker angezeigt wird und was in der Datenbank gespeichert wird.

Ansonsten fehlen mir bisher auch keine Werte, ich fand's nur etwas mühsam herauszufinden, welches Parameter welche Werte enthalten. Einiges ist natürlich intuitiv, einige andere Werte fand/finde ich nicht ganz so intuitiv.

**Reply by akanezumi** [reply]
Ach verdammt. Ich Esel. Negative Werte bedeutet natürlich Einspeisung ins Netz. Das war in den letzten Tagen natürlich wirklich quasi nicht vorhanden. Bitte einfach meine letzten Nachrichten ignorieren.

**Comment by NCIceWolf** [comment]
Denke das Thema ist auch nachhaltig erledigt und kann zu 😃
